### DISCUSSION TITLE
Ability to anchor (skinning) components to the cursor.

### DISCUSSION THREAD

**Discussion by dumbuz** [ppy/osu#22890] [Ideas]
I think the ability to anchor components (like a UR tracker, score, acc, health, etc.) to the cursor instead of just corners of the screen/sides would be pretty cool! Maybe a bit stupid, but stupid in a fun way.

**Comment by AltiDG3** [comment]
I had a similar Idea of adding more anchor points.
Right now when I play mania, I usually only play 4k and so I adjusted the skin that I use for it. However because it is not snapped the skin only works for 4k... I think it be cool if elements can snapped on the mania or taiko board itself.


![osu! 4_19_2023 2_38_50 PM](https://user-images.githubusercontent.com/131309230/233206748-12bc4fed-234a-4d65-b643-163c4be3b836.png)
![osu! 4_19_2023 2_39_25 PM](https://user-images.githubusercontent.com/131309230/233206761-71d209b0-8521-493b-94d2-521c0e0262e3.png)

**Reply by alexpado** [reply]
Hey, look over there, it's getting close ! #24379

**Comment by marioman63** [comment]
Peppy sent out a tweet a couple years ago at this point demonstrating a "bug" where he could anchor elements to the cursor (specifically the input timing bar to the cursor in standard play) which seemed like it would be really useful/fun. Really wish that was kept (would make the input timing bar actually useful if it followed the cursor).
